tests/test_ptycho.py

```diff
@@ -265,9 +272,10 @@ def test_consistent_cgrad(self):
         self.template_consistent_algorithm(
             'cgrad',
             params={
-                'num_gpu': 4,
+                'num_gpu': 1,
                 'recover_probe': True,
                 'recover_psi': True,
+                'use_mpi': False,
```

Comment:
We want to test with `use_mpi = True`! If tests are run without `mpiexec`, it should behave the same as with `use_mpi = False` right? But also, we want to run some of the tests with `mpiexec`.